subxt/src/tx/tx_client.rs

```diff
@@ -224,14 +224,11 @@ where
         Call: TxPayload,
     {
         // Get nonce from the node.
-        let account_nonce = if let Some(nonce) = signer.nonce() {
-            nonce
-        } else {
-            self.client
-                .rpc()
-                .system_account_next_index(signer.account_id())
-                .await?
-        };
+        let account_nonce = self
+            .client
+            .rpc()
+            .system_account_next_index(signer.account_id())
```

Comment:
There still should be some way to set the nonce locally, this will return the next index from the most recent block.

Reply:
e.g. https://github.com/paritytech/smart-bench/blob/361da7782d2eeb72e056b98ce34a2438678e245d/src/wasm/runner.rs#L142

Reply:
Ok didn't know about https://github.com/paritytech/subxt/blob/563afdaded26f954812646f2e163f6e940d50145/subxt/src/tx/tx_client.rs#L128